Metadata.yaml

```diff
@@ -0,0 +1,246 @@
+  it can be applied to problems across discipline boundaries in fluid
+  dynamics, geoscience, plasma physics, nuclear physics, and
+  engineering. It runs at petascale and is designed for future exascale
+  computers.
+
+  SpECTRE is being developed in support of our collaborative Simulating
+  eXtreme Spacetimes (SXS) research program into the multi-messenger
+  astrophysics of neutron star mergers, core-collapse supernovae, and
+  gamma-ray bursts.
+
+Keywords:
+  - Software
+  - Astrophysics
+  - General Relativity
+  - Numerical Relativity
+  - Multiphysics
+  - Gravitational Waves
+  - Discontinuous Galerkin
+  - Finite Difference
+  - Finite Volume
+
+Affiliations:
+  - &aei
+    Max Planck Institute for Gravitational Physics (Albert Einstein Institute),
+    Am Mühlenberg 1, Potsdam 14476, Germany
+  - &caltech
+    Theoretical Astrophysics, Walter Burke Institute for Theoretical
+    Physics, California Institute of Technology, Pasadena, CA 91125, USA
+  - &cornell
+    Cornell Center for Astrophysics and Planetary Science, Cornell University,
+    Ithaca, New York 14853, USA
+  - &fullerton
+    Nicholas and Lee Begovich Center for Gravitational-Wave Physics and
+    Astronomy, California State University, Fullerton, Fullerton, California
+    92831, USA
+  - &new_hampshire
+    Department of Physics & Astronomy, University of New Hampshire, 9 Library
+    Way, Durham NH 03824, USA
+
+Authors:
+  Description: |
+    We maintain three tiers of authors for the code as a whole: Core, Developers
+    and Contributors. The full author list for the SpECTRE DOI is built by
+    concatenating all authors in the order they are listed here.
+
+    The primary goal of this author list is to recognize infrastructure work
+    that does not or cannot have any science papers and results associated with
+    it. Through authorship on the SpECTRE DOI such work gains recognition that
+    holds value in academia. Authorship on the SpECTRE DOI does not directly
+    entitle a person for authorship rights on other publications. Authorship
+    rights on publications related to SpECTRE are governed by our publication
+    policy.
+
+    Note that the placement of authors in these tiers is inherently a subjective
+    topic since we are forced to compare different projects people work on.
+    Nevertheless, having guidelines and some way of promoting healthy and
+    constructive discussions is vital for ensuring the collaborative aspect of
+    the code is healthy.
+
+  Core:
+    Description: |
+      For the time a Developer serves as Core developer they take on some
+      project-management responsibilities, such as merging pull-requests.
+
+      Guidelines on becoming a Core developer:
+
+      - 4-5 major infrastructure contributions, each taking roughly 4-5 weeks of
+        full-time work for an existing Core developer. Or, ∼20-25 weeks of
+        full-time infrastructure work saved for a current Core developer.
```

Comment:
I don't understand this sentence.

Reply:
@nilsdeppe you originally wrote this in the RFC document. Could you clarify what you meant?

Reply:
Removed this sentence for now. If you want me to add a clarified version back in, just let me know.